pipeline/src/node_properties/stream_rtsp_out_properties.rs

```diff
@@ -1,25 +1,26 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use url::Url;
 
-// Ditch all manual Serialize + Deserialize code after changing the properties to specific types.
-// This includes the use of `serialize_with` attribute.
-
-#[derive(Debug, Clone, PartialEq, Serialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct StreamRtspOutProperties {
-    pub uri: Url,
-    #[serde(serialize_with = "super::serialize_required")]
-    pub udp_port: u16,
+    #[serde(flatten)]
+    pub runtime: StreamRtspOutPropertiesRuntime,
```

Comment:
I went with `StreamRtspOutRuntime` pattern in my branch. I think we should either go with that or `StreamRtspOutRuntimeProperties`.

Reply:
`StreamRtspOutRuntime` looks good :+1: